05/css/mobile.css

```diff
@@ -0,0 +1,76 @@
+/* mobile */
+
+.header__container {
+    display: flex;
+    justify-content: space-between;
+    padding: 0;
+}
+.header__logo {
+    min-width: 150px;
+    margin: 5px 5px 5px 80px;
+}
+.header__nav {
+   display: none;
+}
+.content__cell1, .content__cell2 {
+    height: 20vh;
```

Comment:
Lepiej dawać `min-height` bo "coś" może zajmować więcej przestrzeni niż 20vh i wtedy się to rozjedzie ;)